apps/api/drizzle/0040_adorable_random.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE "stake_changes" ALTER COLUMN "delta_amount" SET DATA TYPE numeric(78, 0);--> statement-breakpoint
```

Comment:
this change was also made on a different [PR](https://github.com/recallnet/js-recall/pull/1200/files#diff-56d4bea62ce80a99dab11693ed2fd8cb53fdd9478a405f4a5ec037827315bcb9R18-R19). I'll wait for that one to merge, then I'll remove my migration